src/Windows/Avalonia.Win32/Win32PlatformOptions.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Collections.Generic;
+using Avalonia.OpenGL;
+using Avalonia.Platform;
+
+namespace Avalonia;
+
+public enum Win32RenderingMode
```

Comment:
`Win32RenderingMode` and `Win32CompositionMode` should probably have a zero value ([CA1008](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1008)) so `default` is a valid value. (I know there's no good true default since it depends on the OS version, but `AngleEgl` seems the most appropriate currently).